### ISSUE TITLE
[Question] Angular way and server side processing

### ISSUE THREAD

**Issue by JoniJnm** [l-lin/angular-datatables#836]
Can I use both features?

Is there any demo?

I think that I have to get the data manually, set to $scope, set pagination and render table?

**Comment by milhamj** [comment]
You can add 
`.withOption('fnRowCallback', function (nRow, aData, iDisplayIndex, iDisplayIndexFull) {`
`    		$compile(nRow)($scope);`
`    	})`

to **dtOptions** to make it readable by Angular.

**Comment by l-lin** [comment]
I do not recommend. I advice you use directly with the Ajax renderer.

**Comment by kagant15** [comment]
I have a use case where I need to do a lot of custom rendering in a column (over 25 different dom elements with various parent child relationships) I've already implemented it the angular way when I wasn't loading it with Ajax and it works great.  My problem is the only solution I've found to get my custom rendering accomplished while also loading my data using Ajax is to use DTColumnDefBuilder.renderWith() and return a giant concatenated string with my markup.

How can I customize my markup similar to the angular way while also leveraging server side processing?
